From dfc88c9d983575a2df9f2aa8fd18efd53fdc719a Mon Sep 17 00:00:00 2001
From: Huang Chenyi <b44402@freescale.com>
Date: Mon, 11 Mar 2013 17:18:41 +0800
Subject: [PATCH 172/602] ENGR00253635 Change lcd_density to 160

Change lcd_density to 160, in order to pass the Holo test of CTS

Signed-off-by: Huang Chenyi <b44402@freescale.com>
---
 sabresd_6dq/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index bf91732..adb9f95 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -63,8 +63,8 @@ on boot
     # Set rotation to 270 to cofigure as portrait mode
     setprop ro.sf.hwrotation 0
 
-    # Set the density to 140dpi, default 128dpi is not good
-    setprop ro.sf.lcd_density 140
+    # Set the density to 160dpi, default 128dpi is not good
+    setprop ro.sf.lcd_density 160
 # change for FSL specific service
     chown root system /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
     chmod 0664 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
-- 
1.8.0

